src/App.tsx

```diff
@@ -1,14 +1,69 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { getTodos, getUser } from './api';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [selectedTodoId, setSelectedTodoId] = useState<number | null>(null);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [filteredStatus, setFilteredStatus] = useState('all');
+  const [searchQuery, setSearchQuery] = useState('');
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      const fetchedTodos = await getTodos();
+
+      setTodos(fetchedTodos);
+      setIsLoading(false);
+    };
+
+    loadTodos();
+  }, []);
+
+  const handleShowTodo = async (todo: Todo) => {
```

Comment:
Same here.

You only need async if you have to make an asynchronous request, for example, to get both todos and users in one request. This means you need to wait for todos (await getTodos()) to get everything.